lib/asset-url.js

```diff
@@ -0,0 +1,23 @@
+let postcss = require("postcss");
+let helpers = require("postcss-message-helpers");
+
+module.exports = postcss.plugin("faucet-pipeline-css", ({ manifest }) => {
+	return declarationWalker(node => replaceAssetURL(node, manifest));
+});
+
+function replaceAssetURL(node, manifest) {
+	if(!node.value) {
+		return;
+	}
+
+	node.value = node.value.replace(/asset-url\(['"]?(.*?)['"]?\)/,
```

Comment:
I think this should be wrapped in an `if` to avoid the re-assignments when nothing will change.